boat-scaffold/src/main/java/com/backbase/oss/codegen/angular/BoatAngularGenerator.java

```diff
@@ -184,6 +187,19 @@ public void processOpts() {
                 log.info("  (you can select the angular version by setting the additionalProperty foundationVersion)");
             });
 
+        processOpt(SPEC_VERSION,
+            value -> {
+                if(StringUtils.isNotEmpty(value)) {
+                    additionalProperties.put(SPEC_VERSION, new SemVer(value));
+                }
+            },
+            () -> {
```

Comment:
Again, I'm not sure we should do anything at all when the user does not provide a value.

Reply:
fixed that in main